src/frontend/src/lib/components/tokens/TokensDisplayHandler.svelte

```diff
@@ -16,8 +14,7 @@
 
 	let sortedTokens: TokenUi[];
 	$: sortedTokens = $combinedDerivedSortedNetworkTokensUi.filter(
-		({ id: tokenId }) =>
-			($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n) || displayZeroBalance
+		({ usdBalance }) => (usdBalance ?? 0) || displayZeroBalance
```

Comment:
It's not the same logic anymore - i.e. there is no comparison that the balance is greater than zero - is that expected?

Reply:
The value `0` is typically a `falsy` value, meaning that we can use it as direct condition (and the balance will never be negative, but even if it is, we should probably show it either way)

in any case, to have the scope more clear in the code, i can put the `>0` condition, if you prefer

Reply:
If the behavior remains intact and the change is excepted for sure it's fine.